### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'lang'

### ISSUE THREAD

**Issue by stuartlangridge** [alphacep/vosk-api#982]
Installing and running vosk on Python 3.8 (in Ubuntu 20.04) seems to throw an immediate error with the examples, which looks like either the examples are using an old API parameter or they're using a new API parameter:

```bash
$ python3 -m venv ./venv
$ source ./venv/bin/activate
$ pip3 install vosk
[...]
Successfully installed cffi-1.15.0 pycparser-2.21 vosk-0.3.38

$ git clone https://github.com/alphacep/vosk-api # https://alphacephei.com/vosk/install#usage-examples
[...]
Resolving deltas: 100% (1683/1683), done.
$ cd vosk-api/python/example
$ wget https://alphacephei.com/kaldi/models/vosk-model-small-en-us-0.15.zip
[...]
2022-05-26 10:23:22 (68.6 MB/s) - ‘vosk-model-small-en-us-0.15.zip’ saved [41205931/41205931]
$ unzip vosk-model-small-en-us-0.15.zip
[...]
  inflating: vosk-model-small-en-us-0.15/README  
$ mv vosk-model-small-en-us-0.15 model
$ python3 ./test_simple.py test.wav
Exception ignored in: <function Model.__del__ at 0x7fd1fe987280>
Traceback (most recent call last):
  File ".../venv/lib/python3.8/site-packages/vosk/__init__.py", line 32, in __del__
    _c.vosk_model_free(self._handle)
AttributeError: 'Model' object has no attribute '_handle'
Traceback (most recent call last):
  File "./test_simple.py", line 15, in <module>
    model = Model(lang="en-us")
TypeError: __init__() got an unexpected keyword argument 'lang'

$ python3 --version # debug info as requested
Python 3.8.10
$ pip3 --version
pip 20.0.2 from .../venv/lib/python3.8/site-packages/pip (python 3.8)
$ pip3 -v install vosk
Non-user install because user site-packages disabled
Created temporary directory: /tmp/pip-ephem-wheel-cache-57nw39xl
Created temporary directory: /tmp/pip-req-tracker-z7yxdyym
Initialized build tracking at /tmp/pip-req-tracker-z7yxdyym
Created build tracker: /tmp/pip-req-tracker-z7yxdyym
Entered build tracker: /tmp/pip-req-tracker-z7yxdyym
Created temporary directory: /tmp/pip-install-cmbciqi8
Requirement already satisfied: vosk in .../venv/lib/python3.8/site-packages (0.3.38)
Requirement already satisfied: cffi>=1.0 in .../venv/lib/python3.8/site-packages (from vosk) (1.15.0)
Requirement already satisfied: pycparser in .../venv/lib/python3.8/site-packages (from cffi>=1.0->vosk) (2.21)
Cleaning up...
Removed build tracker: '/tmp/pip-req-tracker-z7yxdyym'
```

**Comment by nshmyrev** [comment]
Please try again, we just published a version 0.3.40 where it should work. Between, there is no need to download model manually now, it is downloaded automatically.

**Comment by stuartlangridge** [comment]
ok! That seems to have fixed the problem, yes.

I did have to `pip3 install requests tqdm` this time: maybe those should be added as dependencies to vosk so that they're downloaded automatically as pycparser and cffi are? But the bug above is fixed!

**Comment by nshmyrev** [comment]
Now fixed in 0.3.41. Thank you!
